server/bin/pbench-verify-indexed.py

```diff
@@ -175,13 +174,23 @@ def main(options):
 
 if __name__ == "__main__":
     parser = ArgumentParser(f"Usage: {_NAME_} [--config <path-to-config-file>]")
+    cfg_name_def = os.environ.get("_PBENCH_SERVER_CONFIG")
     parser.add_argument(
         "-C",
         "--config",
         dest="cfg_name",
-        default=os.environ.get("_PBENCH_SERVER_CONFIG"),
+        default=cfg_name_def,
         help="Specify config file",
```

Comment:
The local `cfg_name_def` variable doesn't seem like it's worth creating.

Reply:
Left over from a failed experiment.